Marlin/example_configurations/K8400/DUAL-HEADS-Configuration.h

```diff
@@ -0,0 +1,1216 @@
+/**
```

Comment:
Please keep the name `Configuration.h` and just use sub-folders for the different variants.